src/index.js

```diff
@@ -2,6 +2,13 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './styles/index.css';
 import App from './components/App';
+import { Router, Route, hashHistory } from 'react-router';
+import LoggedUserPage from "./components/LoggedUserPage";
 
 
-ReactDOM.render(<App/>, document.getElementById('root'));
```

Comment:
It would be better if you try to encapsulate the Router inside the App component.